src/main/java/http/response/BuildResponse.java

```diff
@@ -0,0 +1,26 @@
+package http.response;
+
+import java.util.*;
+
+public class BuildResponse {
```

Comment:
all of these functions make sense and are tested very well. I just wonder - what do you think about having one function - `buildResponse` that would use `buildStartLine`, `parseHeaderLine` and `buildHeaderLine` internally and these 3 will become private?

Reply:
Thanks justyna, I agree with this I'll make the change 👍